coala_quickstart/generation/Utilities.py

```diff
@@ -68,12 +68,13 @@ def get_gitignore_glob(project_dir, filename=".gitignore"):
         A list generator of glob expressions generated from the
         ``.gitignore`` file.
     """
-    gitignore = os.path.join(project_dir, filename)
-
-    with open(gitignore, "r") as file:
-        for line in file:
-            for glob in parse_gitignore_line(line):
-                yield os.path.join(project_dir, glob)
+    for dirName, subdirList, fileList in os.walk(project_dir):
```

Comment:
Not required if you collect all the paths at https://github.com/coala/coala-quickstart/pull/110/files#diff-7e6eaad4d79d2cfe0c15b00a0c204498R29 and pass them as an argument.